schemas/consensus_sequence.json

```diff
@@ -284,7 +284,8 @@
                 "Missing",
                 "Not Collected",
                 "Not Provided",
-                "Restricted Access"
+                "Restricted Access",
+                "Pharynx (throat)"
```

Comment:
@edsu7 just a very minor comment. I noticed that other than missing values, all other values are sorted in alphabet order.